docs/pagination.md

```diff
@@ -36,6 +36,36 @@ testdata:
 ```
 {% endraw %}
 
+Or this one in JavaScript, in case you prefer another [templating language](/languages/):
+
+{% codetitle "JavaScript", "Syntax" %}
+
+{% raw %}
+```js
+exports.data = {
+  pagination: {
+    data: "testdata",
+    size: 2
+  },
+  testdata: [
+    "item1",
+    "item2",
+    "item3",
+    "item4"
+  ]
+};
+
+exports.render = function(data) {
+  return `<ol>
+    ${data.pagination.items.map(function(item) {
+        return `<li>${item}</li>`;
+      }).join('');
```

Comment:
These should be double quotes inside the `join("")`. I promise, I’ll get the hang of this style guide.